packages/x-charts/src/models/seriesType/line.ts

```diff
@@ -60,6 +60,11 @@ export interface LineSeriesType
    * The label to display on the tooltip or the legend. It can be a string or a function.
    */
   label?: string | ((location: 'tooltip' | 'legend') => string);
+  /**
+   * The type of curve to use for the line. Read more about curves at
+   * [line interpolation](https://mui.com/x/react-charts/lines/#interpolation).
```

Comment:
The line break on API doc seems a bit strange. 🙈 
<img width="609" alt="Screenshot 2024-10-01 at 17 17 13" src="https://github.com/user-attachments/assets/07691a8b-3fa1-406b-aabe-53c32bb4983e">

Maybe going with this would produce a cleaner result?
```suggestion
   * The type of curve to use for the line.
   * Read more about curves at [line interpolation](https://mui.com/x/react-charts/lines/#interpolation).
```